src/app/roadmap/roadmap.json

```diff
@@ -87,7 +87,40 @@
       "tracking_issue": "https://github.com/n0-computer/iroh/issues/2842"
     },
     { "version": "v0.30.0", "done": true, "released": "2024-12-16", "doc": "https://www.iroh.computer/blog/iroh-0-30-0-slimming-down" },
+    {
+      "done": true,
+      "title": "Auth on Relays",
+      "description": "Blacklist or whitelist `NodeId`s on the relay",
+      "tracking_issue": "https://github.com/n0-computer/iroh/pull/3086"
+    },
+    {
+      "done": true,
+      "title": "Iroh Doctor CLI v0.1",
+      "description": "release the first version of the iroh doctor CLI as a standalone tool for diagnosing and reporting on connectivity",
+      "tracking_issue": null
+    },
+    { "version": "v0.31.0", "done": true, "released": "2024-01-14", "doc": "https://www.iroh.computer/blog/iroh-0-31-0-back-to-fighting-fit" },
+    {
+      "done": true,
+      "title": "Add QUIC Address Discovery to `iroh`",
+      "description": "iroh can launch QUIC Address Discovery probes",
+      "tracking_issue": "https://github.com/n0-computer/iroh/pull/3049"
+    },
+    { "version": "v0.32.0", "done": true, "released": "2024-02-03", "doc": "https://www.iroh.computer/blog/iroh-0-32-0-browser-alpha-qad-and-n0-future" },
     { "ellipsis": true },
+    {
+      "done": false,
+      "title": "browser support",
+      // "description": "iroh compiles to web assembly and runs in the browser",
```

Comment:
I suspect it's due to this comment.
I'll check by running `npm run dev` and going to the roadmap locally, though.

Reply:
Yeah - from running `npm run dev` locally:
> Module parse failed: Cannot parse JSON: Expected double-quoted property name in JSON at position 4737 while parsing '{